compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -177,11 +179,18 @@ printTestCoverage ShowCoverage {getShowCoverage} extPkgs modules results
     pkgIdToPkgName pId = maybe pId LF.unPackageName $ join $ M.lookup pId pkgMap
     templates = [(pidM, m, t) | (pidM, m) <- modules, t <- NM.toList $ LF.moduleTemplates m]
     choices =
-        [ (pidM, m, t, n)
+        [ (pidM, m, t, n, mi)
         | (pidM, m, t) <- templates
-        , n <- concat
-            [ NM.names (LF.tplChoices t)
-            , S.toList . LF.tpiInheritedChoiceNames =<< NM.toList (LF.tplImplements t)
+        , (n, mi) <- concat
+            [ [ (choice, Nothing)
```

Comment:
@robin-da, do you find this version easier to follow?

Reply:
yes, that looks much better to me.thanks!